src/packages/media/media/components/input-image-cropper/image-cropper-focus-setter.element.ts

```diff
@@ -9,36 +12,48 @@ export class UmbImageCropperFocusSetterElement extends LitElement {
 	imageElement!: HTMLImageElement;
 
 	@query('#wrapper')
-	wrapperElement?: HTMLImageElement;
+	wrapperElement?: HTMLElement;
 
 	@query('#focal-point')
-	focalPointElement!: HTMLImageElement;
+	focalPointElement!: HTMLElement;
+
+	@state() private isDraggingGridHandle = false;
+
+	@state() private coords = { x: 0, y: 0 };
 
 	@property({ attribute: false })
-	focalPoint: UmbImageCropperFocalPoint = { left: 0.5, top: 0.5 };
+	get focalPoint() {
+		return this.#focalPoint;
+	}
+	set focalPoint(value) {
+		this.#focalPoint = value;
+		this.#onFocalPointUpdated();
+	}
+
+	#focalPoint: UmbImageCropperFocalPoint = { left: 0.5, top: 0.5 };
 
 	// TODO: [LK] Temporary fix; to be reviewed.
 	@property({ type: Boolean })
 	hideFocalPoint = false;
+	
+	@property({ type: Boolean, reflect: true }) disabled = false;
 
 	@property({ type: String })
 	src?: string;
 
-	#DOT_RADIUS = 6 as const;
+	#DOT_RADIUS = 8 as const;
 
 	override disconnectedCallback() {
 		super.disconnectedCallback();
-		this.#removeEventListeners();
 	}
 
 	protected override updated(_changedProperties: PropertyValueMap<any> | Map<PropertyKey, unknown>): void {
 		super.updated(_changedProperties);
 
 		if (this.hideFocalPoint) return;
 
-		if (_changedProperties.has('focalPoint') && this.focalPointElement) {
-			this.focalPointElement.style.left = `calc(${this.focalPoint.left * 100}% - ${this.#DOT_RADIUS}px)`;
-			this.focalPointElement.style.top = `calc(${this.focalPoint.top * 100}% - ${this.#DOT_RADIUS}px)`;
+		if (_changedProperties.has('focalPoint') && this.focalPoint) {
+			this.#setFocalPointStyle(this.focalPoint.left, this.focalPoint.top);
```

Comment:
Since there is now a setter for `focalPoint`, consider removing the `updated` hook and integrating it into the that setter.

Reply:
So move `#setFocalPointStyle()` insider setter for `focalPoint`?

Reply:
Yes, I thought if it just needs to react on the changes to `focalPoint`, then it may as well be in the setter of that and we can remove this whole method.

Reply:
This has been updated as well.